src/test/scala/mesosphere/marathon/tasks/InstanceOpFactoryHelperTest.scala

```diff
@@ -51,10 +51,11 @@ class InstanceOpFactoryHelperTest extends UnitTest {
       val taskInfo = MarathonTestHelper.makeOneCPUTask(task.taskId).build()
 
       When("We create a launch operation")
-      val launch = f.helper.provision(taskInfo, task, instance)
+      val now = Timestamp.now()
+      val launch = f.helper.provision(taskInfo, instance.instanceId, instance.agentInfo.get, instance.runSpecVersion, task, now)
 
       Then("The result is as expected")
-      launch.stateOp shouldEqual InstanceUpdateOperation.Provision(instance)
+      launch.stateOp shouldEqual InstanceUpdateOperation.Provision(instance.instanceId, instance.agentInfo.get, instance.runSpecVersion, Seq(task), now)
```

Comment:
How about using Scala test's `inside` here?

Reply:
I don't like that too much, I find the errors that are coming out of that cryptic. I think this is pretty readable, I would stick with this implementation

Reply:
Hehe, that's funny. I actually do not like these long stings dumps of X was not Y here are 20 lines with 120 characters each to check for you 😉. We can implement nicer diffs in the future. That should not block this PR.